### PR TITLE
Update instruction copy

### LINKED ISSUE #9517: Beta shop says I don't need to attach machines to set up UA services
When I go to the beta shop at https://ubuntu-com-9240.demos.haus/advantage, it says "You don’t need to attach machines before setting up UA services." for Ubuntu 16.04 and 18.04.

Screenshot from 2021-04-07 12-06-34

Is this because the UA client hasn't been officially released for these series yet?

Perhaps this will be addressed soon, but I would expect this to either link to documentation on how to actually set up access to services with the "old" UA client, or to assume that the new UA client is already released in this copy, if it is expected to be released along with the beta shop.

I think it is misleading as it is though; as a customer I would read this and think that 16.04 and 18.04 automatically enable all the UA services with no special instructions required -- which is not the case.

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Update instruction copy for 16.04 and 18.04 according to https://github.com/canonical-web-and-design/ubuntu.com/issues/9517#issuecomment-820183733

## QA

-  https://ubuntu-com-9669.demos.haus/advantage?test_backend=true

## Issue / Card

Fixes #9517

## Screenshots

image

### CODE REVIEW

**Comment by ClementChaumel** [comment]
@lechsandecki here are the changes to the instructions. Please review it and provide instruction for the 16.04 and 18.04 changes thanks!

File: templates/advantage/table/_attached-machines.html [hunk 1]
```diff
@@ -3,32 +3,49 @@
...
+              <span>Add the PPA</span>
+              <code>sudo add-apt-repository ppa:ua-client/stable<br/>
+              sudo apt-get update</code>
+              <hr />
+              <span>Install the UA client</span>
+            <code>sudo apt install ubuntu-advantage-tools</code>
+            </div>
+          </div>
+        <div class="advantage-table-section">
+          <p class="u-no-margin--bottom">Once installed or if you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
```
Comment (lechsandecki) [inline templates/advantage/table/_attached-machines.html:19]: I'd drop that. I suggest that, on this page, we keep the following:

to attach a machine: sudo ua abc
to detach a machine: sudo ua def

Please ensure that you have the latest version of the ua client installed: apt-get upgrade

File: templates/advantage/table/_attached-machines.html [hunk 2]
```diff
@@ -3,32 +3,49 @@
...
```
Comment (lechsandecki) [inline templates/advantage/table/_attached-machines.html:48]: we also need to update the ESM screen (it's now 14.04 or 16.04, not just 14.04)
Similarly, Livepach can get activated via the ua client for each Ubuntu LTS now. We don't need separate "Ubuntu 16.04 LTS and 18.04 LTS" instructions anymore

**Review by lechsandecki — CHANGES_REQUESTED** [review]

**Comment by albertkol** [comment]
@ClementChaumel you broke the table

image

**Comment by ClementChaumel** [comment]
I've updated the copy according to your review @lechsandecki, please take another look:
image
image
image

and I fixed the table @albertkol :angel:

**Comment by lechsandecki** [comment]
+1, looks great. Can we make it live asap, please?

**Comment by SirSamTumless** [comment]
Hi @lsandecki I don't think it to say "we are slow implementation" in the public channnel. We ask for your review yesterday and you've only just come abck.

**Review by lechsandecki — APPROVED** [review]

**Pull request merged**